src/thing_traps.h

```diff
@@ -63,7 +65,7 @@ enum TrapActivationTypes {
     TrpAcT_Power
 };
 
-struct Thing;
+struct Thing;//what is this?
```

Comment:
a forward declaration

Reply:
if it compiles without it it's not needed, but basically used to tell compiler Things exist